Makefile

```
@@ -47,6 +47,8 @@ serve:
 
 pipeline: fetch build serve
 
+run: build serve
```

checkmake (0.3.2)

[warning] Target "run" should be declared PHONY.

(phonydeclared)